src/layerstack.stanza

```diff
@@ -355,10 +355,12 @@ When used on its own, this function is useful for creating asymmetric
 pcb stackups. This function is most useful when used on a non-empty
 layer stack.
 
-@param l-set layer or set of layers to add to the top of the stack. Note
+@param l-set layer or set of layers to add to the top of the stack, ordered from inner to outer (bottom to top). Note
```

Comment:
Let's please remove `(bottom to top)` - this is confusing because we also talk about the bottom side of the board vs the top side of the board. 

I would be OK with `ordered from inner-most to outer -most on the top side`